src/deroff.rs

```diff
@@ -1001,6 +1002,8 @@ impl Deroffer {
     }
 
     fn flush_output<W: std::io::Write>(&mut self, mut write: W) {
+        println!("{}", self.get_output());
```

Comment:
Oh, is this needed? Shouldn't we write to writer instead of stdout?

Reply:
Oops that was a debugging thing  from a WHILE ago. fixed